.github/workflows/run-tests.yml

```
@@ -0,0 +1,32 @@
+name: Run tests
+
+on:
+  - push
```

Do we need to run all tests for every single push? Let's run for merges to the main branch only

```suggestion
  - push:
      branches: [main]
```

---

+1